src/test/java/net/sf/jabref/importer/fileformat/SilverPlatterImporterTest.java

```diff
@@ -0,0 +1,85 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+
+public class SilverPlatterImporterTest {
+
+    @Before
+    public void setUp() throws Exception {
+        if (Globals.prefs == null) {
+            Globals.prefs = JabRefPreferences.getInstance();
+        }
+    }
+
+    @Test
+    public final void testIsRecognizedFormat() throws Exception {
+        SilverPlatterImporter importer = new SilverPlatterImporter();
+        //test with recognized format
+        try (InputStream stream = SilverPlatterImporterTest.class
+                .getResourceAsStream("SilverPlatterImporterTest1.txt")) {
+            Assert.assertTrue(importer.isRecognizedFormat(stream));
+        }
+
+        //test with not recognized format
```

Comment:
Split the test into two tests (one for recognized formats, one for not recognized)